qlib/backtest/exchange.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         close_cost=0.0025,
         min_cost=5,
         extra_quote=None,
-        quote_cls=PandasQuote,
+        quote_cls=NumpyQuote,
```

Comment:
If NumpyQuote is not for general purpose, it should be configurable and not the default value.